cmd/mysterium_node/mysterium_node.go

```diff
@@ -89,3 +95,52 @@ func NewCommand() (*cli.App, error) {
 
 	return app, nil
 }
+
+func newApp() (*cli.App, error) {
+	app := cli.NewApp()
+	return app, config.RegisterFlagsNode(&app.Flags)
+}
+
+// uiCommands is a map which consists of all
+// commands are used directly by a user.
+var uiCommands = map[string]struct{}{
+	command_cli.CliCommandName: {},
+}
+
+// readyOnceFunc returns a func which is only run once and can
+// be executed to preconfigure global settings used in the app.
+func readyOnceFunc() cli.BeforeFunc {
```

Comment:
I think this is a terrible name, `configureCLILogging` even just `configureLogging` is better and would reduce comment docs by a whole line